src/extension/op_def.rs

```diff
@@ -256,106 +276,78 @@ impl OpDef {
     }
 }
 
-impl Extension {
-    /// Add an operation definition to the extension.
-    fn add_op(
+/// Wrapper to allow editing an OpDef before it is added to an extension.
```

Comment:
Do you really mean "before"? Or, you could make `add_op` return the OpDefMut (and be the only way to do so); you'll have to give up the OpDefMut when you move the Extension into an ExtensionRegistry....

Reply:
OpDefMut removed using `Arc::get_mut`